modules/i18n/input-method/fcitx5.nix

```
@@ -18,6 +18,14 @@ in {
           Enabled Fcitx5 addons.
         '';
       };
+      waylandFrontend = mkOption {
```

I suggest always putting a space between options, otherwise it is a bit tricky to tell them apart.

```suggestion

      waylandFrontend = mkOption {
```